.github/workflows/test_fatjar_cli.yml

```diff
@@ -0,0 +1,29 @@
+name: Test CLI FatJar
+on:
+  push:
+    branches:
+      - main
+      - release-*
+  pull_request:
+    branches:
+      - main
+      - release-*
+jobs:
+  verify:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Download drivers
+        run: scripts/download_driver_for_all_platforms.sh
+      - name: Intall Playwright
+        run: mvn install -D skipTests --no-transfer-progress
+      - name: Test CLI version
+        shell: bash
+        run: tools/test-cli-farjar-version/test.sh
```

Comment:
Let's just merge this step into `.github/workflows/test_cli.yml` to avoid extra download & build of playwright on CI just for this test, we already do this for other cli tests.